README.md

```diff
@@ -34,7 +34,23 @@ Supports popular public checkpoints such as:
 
 This implementation extends on [Lit-LLaMA](https://github.com/lightning-AI/lit-llama) and [nanoGPT](https://github.com/karpathy/nanoGPT), and it's **powered by [Lightning Fabric](https://lightning.ai/docs/fabric/stable/) ⚡**.
 
-## Design principles
+
+
+---
+
+**NeurIPS 2023 Large Language Model Efficiency Challenge: 1 LLM + 1GPU + 1Day**
+
+Lit-GPT is the official starter kit of the [NeurIPS 2023 LLM efficiency challenge](https://llm-efficiency-challenge.github.io), which is focused on tackling the three main challenges of LLMs: 1) transparency, 2) standard benchmarks, and 3) hardware access. 
+
+If you are interested in participating, you can learn more about the official NeurIPS LLM efficiency challenge website [here](https://llm-efficiency-challenge.github.io). **The submission deadline is Oct 15th, 2023.**
+
+---
```

Comment:
Depending on how big is the deal that `Lit-GPT` is the official starter-kit for the challenge (because I have no idea how serious you guys about it) maybe it's worth to highlight it even more?
> **Note**
...


or even

> **Important**
....

what do you think?